en/docs/learn/examples/jms-examples/specifying-a-delivery-delay-on-messages.md

```diff
@@ -29,23 +29,20 @@ See the instructions on how to [build and run](#build-and-run) this example.
                         <property name="messageType" scope="axis2" value="application/xml"/>
                         <property action="remove" name="Content-Type" scope="transport"/>
                         <log level="full"/>
-                        <send>
+                        <call>
                             <endpoint>
                                 <address uri="jms:transport.jms.ConnectionFactory=myQueueConnectionFactory"/>
                             </endpoint>
-                        </send>
+                        </call>
+                        <respond/>
                     </inSequence>
-                    <outSequence>
-                        <send/>
-                    </outSequence>
                 </target>
-                <publishWSDL uri="file:repository/samples/resources/proxy/sample_proxy_1.wsdl"/>
+                <publishWSDL uri="file:repository/samples/resources/proxy/sample_proxy_1.wsdl" preservePolicy="true"/>
```

Comment:
Is this preservePolicy  attribute required?

Reply:
Yes. It would give an error "Attribute 'preservePolicy' is missing from element 'publishWSDL'" if it is not provided.